diff -r 70287b5cb024 lib_pypy/dbm.py
--- a/lib_pypy/dbm.py	Tue Sep 27 11:26:38 2016 +0200
+++ b/lib_pypy/dbm.py	Wed Sep 28 22:43:14 2016 +0000
@@ -127,7 +127,7 @@
     if not libpath:
         # XXX this is hopeless...
         for c in ['5.3', '5.2', '5.1', '5.0', '4.9', '4.8', '4.7', '4.6', '4.5']:
-            libpath = ctypes.util.find_library('db-%s' % c)
+            libpath = ctypes.util.find_library('gdbm-%s' % c)
             if libpath:
                 break
         else:
